Sources/Bot/Services/MergeService.swift

```diff
@@ -234,7 +237,7 @@ extension MergeService {
                         case .pending:
                             return .value(.integrationDidChangeStatus(.updating, metadata))
                         case .failure:
-                            return .value(.integrationDidChangeStatus(.failed(.checkingCommitChecksFailed), metadata))
+                            return .value(.integrationDidChangeStatus(.failed(.checksFailing), metadata))
```

Comment:
This was a typo I discovered while working on the codebase, figured I'd fix it while I was at it